src/SDK/Language.php

```diff
@@ -108,7 +108,6 @@ protected function toCamelCase($str): string
 
         $str = \preg_replace('/[^a-zA-Z0-9]+/', ' ', $str);
         $str = \trim($str);
-        $str = strtolower($str);
```

Comment:
how is this relevant for nested serialization?

Reply:
This was stacked from the other PR